modules/configuring-ibm-cos.adoc

```diff
@@ -0,0 +1,138 @@
+
+You create an {ibm-cloud-object-storage} instance to store the {oadp-short} backup data. After you create the COS instance, you configure the COS instance with `HMAC` service credentials.
+
+.Prerequisites
+
+* You have an {ibm-cloud-title} Platform account.
+* You installed the link:https://cloud.ibm.com/docs/cli?topic=cli-getting-started[{ibm-cloud-title} CLI].
+* You are logged in to {ibm-cloud-title}.
+
+.Procedure
+
+. Install the {ibm-cloud-object-storage} plugin by running the following command. For more information, refer link:https://cloud.ibm.com/docs/cli?topic=cli-plug-ins[{ibm-cloud-title} plugin].
++
+[source,terminal]
+----
+$ ibmcloud plugin install cos -f
+----
+
+. Set a bucket name.
++
+[source,terminal]
+----
+$ BUCKET=<bucket_name>
+----
+
+. Set a bucket region.
++
+[source,terminal]
+----
+$ REGION=<bucket_region> <1>
+----
+<1> For example, `eu-gb`.
+
+. Create a resource group by running the following command:
++
+[source,terminal]
+----
+$ ibmcloud resource group-create <resource_group_name>
+----
+
+. Set the `target` resource group by running the following command:
++
+[source,terminal]
+----
+$ ibmcloud target -g <resource_group_name>
+----
+
+. Verify that the target resource group is correctly set, by running the following command. In the example output, the resource group is set to `Default`.
++
+[source,terminal]
+----
+$ ibmcloud target
+----
++
+.Example output
++
+[source,yaml]
+----
+API endpoint:     https://cloud.ibm.com
+Region:           
+User:             test-user
+Account:          Test Account (fb6......e95) <-> 2...122
+Resource group:   Default
+----
+
+. Set the resource group variable.
++
+[source,terminal]
+----
+$ RESOURCE_GROUP=<resource_group> <1>
+----
+<1> For example, `"default"`.
+
+. Create an {ibm-cloud-title} resource `service-instance` by running the following command. For more details, refer link:https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-provision[Choosing a plan and creating an instance].
++
+[source,terminal]
+----
+$ ibmcloud resource service-instance-create \
+<service_instance_name> \// <1>
+<service_name> \// <2>
+<service_plan> \// <3>
+<region_name> // <4>
+----
+<1> Specify a name for the `service-instance`.
+<2> Specify the service name. Alternatively, you can also specify a service ID instead of the service name.
+<3> Specify the service plan for your {ibm-cloud-title} account.
+<4> Specify the region name. 
++
+.Example command
++
+[source,terminal]
+----
+$ ibmcloud resource service-instance-create test-service-instance cloud-object-storage \ // <1>
+standard \
+global \
+-d premium-global-deployment // <2>
+----
+<1> `cloud-object-storage` is the service name.
+<2> The `-d` flag specifies the deployment name.
+
+. Extract the `service-instance-id` by running the following command:
++
+[source,terminal]
+----
+$ SERVICE_INSTANCE_ID=$(ibmcloud resource service-instance test-service-instance --output json | jq -r '.[0].id')
```

Comment:
Keeping it as jq for now. The alternative is to add an extra step to instruct the user to make a note of the service instance id from the previous step. Discussed with sme, instead of copy paste, the current command is a better option for now. Thanks for the suggestion. I will keep this in mind for the next time.